src/network-uptime/requirements.txt

```diff
@@ -2,5 +2,6 @@ numpy
 pandas
 matplotlib
 requests
+datetime
```

Comment:
@uman95 , you might consider adding version numbers to these packages, we had issues with some dependencies.

Reply:
@sserurich done. Thanks.